src/Microsoft.DotNet.Build.Tasks/Targets/post-process-assembly.targets

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <IntermediateAssemblyPath>$(IntermediateOutputPath)$(TargetName)$(TargetExt)</IntermediateAssemblyPath>
+    <PostProcessAssemblyMarker>$(IntermediateAssemblyPath).postprocessed</PostProcessAssemblyMarker>
+  </PropertyGroup>
+
+  <Target Name="PostProcessAssembly"
+    AfterTargets="CoreCompile"
+    Inputs="$(IntermediateAssemblyPath)"
+    Outputs="$(PostProcessAssemblyMarker)">
+
+    <CallTarget Targets="@(PostProcessAssemblySteps)" />
```

Comment:
> but if the seed or contract changes you also need to rerun it. 

At least for partial facades, you are going to need to start over from scratch if you want to re-direct the type forwards that have been added. There's no logic in GenFacades for modifying existing type-forwards right now. I think this might indicate that we actually need to include the seed/contract assembly as inputs to CoreCompile if we want truly correct incremental builds, as we need to rebuild the whole assembly to re-do type forwards.

In that case, it might make sense to just hook FillPartialFacade directly after Corecompile with no Inputs/Outputs (basically what Wes said) and undo my changes to sign.targets.